README.md

```diff
@@ -1,48 +1,27 @@
-![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
+![TPI](https://static.iterative.ai/img/cml/banner-terraform.png)
 
-# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+# Terraform Provider Iterative
 
-The Iterative Provider is a Terraform plugin that enables full lifecycle
-management of computing resources for machine learning pipelines, including GPUs, from your favorite cloud vendors.
+[![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
 
-The Iterative Provider makes it easy to:
+- **Orchestrate Resources**: create cloud compute & storage resources without reading pages of documentation
```

Comment:
> _without reading pages of documentation_

[This is not a pipe](https://en.wikipedia.org/wiki/The_Treachery_of_Images)

Reply:
```suggestion
- **Orchestrate Resources**: create cloud compute & storage resources without reading eldritch scrolls of documentation
```

Reply:
Looks good to me! 🧝🏼‍♂️📜